app/assets/tailwind/application.css

```
@@ -1,6 +1,80 @@
 @import "tailwindcss";
 @plugin "daisyui";
 
+/*! Yaku Han JP v4.1.1 (OFL-1.1 AND MIT) by Qrac | Type: YakuHanRPs - Based on M PLUS Rounded 1c */
+@font-face {
+  font-family: "YakuHanRPs";
```

Stylelint (17.14.0)

[error] Expected no quotes around "YakuHanRPs" (font-family-name-quotes)

(font-family-name-quotes)